### DISCUSSION TITLE
How does FAST deal with the airfoil lift and drag coefficients under other Reynolds numbers?

### DISCUSSION THREAD

**Discussion by Zhiming-Cai** [OpenFAST/openfast#2543] [General]
Dear all,
I have a question and I would like to ask you, the airfoil lift and drag coefficients in FAST are only defined under one Reynolds number (for example, nrel5mw is defined under a Reynolds number of 750,000), but the Reynolds number in the environment will also change after setting the inflow change, so how does FAST deal with the airfoil lift and drag coefficients under other Reynolds numbers?
@NRELAdmin

**Comment by jjonkman** [comment]
Dear @CaiZhiming-HHU,

The AeroDyn module of OpenFAST does support Reynold's number interpolation if your airfoil data contains multiple tables (`NumTabs` > 1) at different Reynolds numbers (`Re`) and you've selected `AFTabMod` = 2 in AeroDyn.  See the MHK_RM1_Fixed case in the OpenFAST r-test for an example: https://github.com/OpenFAST/r-test/tree/main/glue-codes/openfast/MHK_RM1_Fixed.

If you've set `AFTabMod` = 1 and only have one table for a given airfoil, AeroDyn will use that data regardless of the local Reynolds number.

Best regards,

**Comment by RBergua** [comment]
OpenFAST (AeroDyn) is certainly able to interpolate the airfoil data (e.g., lift and drag coefficients) based on the Reynolds number. This capability was used in the OC6 Phase III project. See for example Figure 4 and Table 4 here: https://www.nrel.gov/docs/fy23osti/83102.pdf. Taking advantage of the AFTabMod parameter in the *primary AeroDyn input file*, the user can allow the 2D interpolation based on Re:
![image](https://github.com/user-attachments/assets/7b9d0b2c-c1ea-406f-b28b-b86d5d75dbfb)
Then in the airfoil input file, you should define:
![image](https://github.com/user-attachments/assets/10c6802e-cec5-4f98-bc4d-8c30718c248b)
I hope that helps!

**Comment by Zhiming-Cai** [comment]
Ok, I understand. Thank you both for your answers. This is a great help to me.
